webAO/client.js

```diff
@@ -870,13 +884,20 @@ class Client extends EventEmitter {
 	 * in one packet.
 	 * @param {Array} args packet arguments
 	 */
-  handleSC(args) {
+  async handleSC(args) {
+    const sleep = (ms) => new Promise((r) => setTimeout(r, ms));
+
+    // Add this so people can see characters loading on the screen.
+    document.getElementById('client_loading').style.display = 'none';
+    document.getElementById('client_charselect').style.display = 'block';
+
     document.getElementById('client_loadingtext').innerHTML = 'Loading Characters';
     for (let i = 1; i < args.length; i++) {
       document.getElementById('client_loadingtext').innerHTML = `Loading Character ${i}/${this.char_list_length}`;
       const chargs = args[i].split('&');
       const charid = i - 1;
-      setTimeout(() => this.handleCharacterInfo(chargs, charid), charid * 10);
+      await sleep(0.1); // TODO: Too many network calls without this. net::ERR_INSUFFICIENT_RESOURCES
```

Comment:
i thought too many pending requests causes ERR_INSUFFICIENT_RESOURCES - so spreading them out for a bit longer would only make it take that much longer before you experience the issue again, wouldn't it? if this issue resurfaces, what if you cap the number of concurrent requests being made with something like [`async.queue`](https://caolan.github.io/async/v3/docs.html#queue)?